grid_helper.js

```diff
@@ -505,7 +534,7 @@ function smartdownLoaded() {
     buildColumnLayout(debugContent.length, numColumns) :
     null;
   // debugLayout = buildStaggeredLayout(3, numColumns);
-  debugLayout = null;
+  //debugLayout = null;
```

Comment:
I think that when committing the PR, it'd be nice if it wasn't in super-debug mode. `authorMode=true` is fine and good for demo, but the Debug content is probably not that useful and will go eventually go away, so disable it by uncommenting the line: `debugLayout = null;`